pkgs/development/python-modules/scikit-network/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  cython,
+  setuptools,
+  numpy,
+  scipy,
+  pytest-runner,
+  pytest,
+  nose,
+  pluggy,
+}:
+buildPythonPackage rec {
+  pname = "scikit-network";
+  version = "0.33.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "scikit-network";
+    hash = "sha256-9/fMAutCZ7Fqk4P2RiYVLlm6Q11td7GjVmW2/BvPVy0=";
+  };
+
+  # can be removed if https://github.com/sknetwork-team/scikit-network/pull/581 is merged
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail "__builtins__.__NUMPY_SETUP__ = False" "__builtins__['__NUMPY_SETUP__'] = False"
+  '';
+
+  build-system = [
+    cython
+    setuptools
+    numpy
+    scipy
+    pytest-runner
```

Comment:
Should go into `nativeCheckInputs` IMHO, even if it is listed as setup requirement for _some_ reason by scikit-network.

Reply:
We do not need `pytest-runner` because we run tests using `pytestCheckHook`.

> Setup scripts can use pytest-runner to add setup.py test support for pytest runner.
https://pypi.org/project/pytest-runner/

Reply:
Right, thanks! I seldomly do python packaging, just see it in the list of some other packages. :+1: